src/anupSJun20/StringReplace.java

```diff
@@ -0,0 +1,51 @@
+package anupSJun20;
+/*Replace second last globant with Technocredits. 
+input-> Hi Globant Hello Globant Hi Globant GM Globant Hi Globant
+
+output->  Hi Globant Hello Globant Hi Globant GM Technocredits Hi Globant*/
+
+import java.util.Arrays;
+
+public class StringReplace {
+	String findTheNewString(String str, String replace) {
+		String[] newstr = str.split(" ");
+		int count = 0;
+		String finalString = "";
+		for(int index = 0;index<newstr.length;index++) {
+			if(newstr[index].equals(replace)) {
+				count++;
```

Comment:
Hi anup,
Line number 28 th logic can be placed here
if(newcount == 2) {
				newstr[index] = "Technocredits";
				break;
			}

So this will save your one for loop Time.

Reply:
Thank you Harsh for the feedback. Updated the code.